blob/fileblob/fileblob.go

```diff
@@ -565,6 +582,20 @@ func (b *bucket) Delete(ctx context.Context, key string) error {
 }
 
 func (b *bucket) SignedURL(ctx context.Context, key string, opts *driver.SignedURLOptions) (string, error) {
-	// TODO(Issue #546): Implemented SignedURL for fileblob.
+	path, err := b.path(key)
+	if err != nil {
+		return "", err
+	}
+
+	return b.urlSigner.SignedURL(ctx, key, path, opts)
+}
+
+type URLSigner interface {
+	SignedURL(ctx context.Context, key string, path string, opts *driver.SignedURLOptions) (string, error)
```

Comment:
It would be good to give this method a different name, since it is a different signature than the bucket driver's `SignedURL` method now. There has been confusion in Go Cloud in the past with methods that have identical names but ever-so-slightly different functionality.

Perhaps `SignedFileURL`? If you're on board with that, I don't think the interface name has to change, as the full qualified identifier (`fileblob.URLSigner`) already contains the word "file".